common/beerocks/bwl/include/bwl/nl80211_client.h

```diff
@@ -33,6 +51,116 @@ namespace bwl {
 class nl80211_client {
 
 public:
+    /**
+     * @brief Channel information
+     *
+     * Used in sBandInfo structure as each band contains a list of supported channels.
+     *
+     * Information obtained with NL80211_CMD_GET_WIPHY command through a NL80211 socket.
+     * See NL80211_FREQUENCY_ATTR_* in <linux/nl80211.h> for a description of each field.
+     */
+    struct sChannelInfo {
+
+        /**
+         * Channel number.
+         * Obtained from NL80211_FREQUENCY_ATTR_FREQ (see 802.11-2007 17.3.8.3.2 and Annex J)
+         */
+        uint16_t channel = 0;
+
+        /**
+         * Maximum channel bandwidth in MHz.
+         * Possible values are 0 (unknown), 20, 40, 80 and 160 MHz.
+         */
+        uint16_t bandwidth_MHz = 0;
```

Comment:
Please don't mix camel case and lower snake case. The comment is enough actually so its fine to just call it `bandwidth`
```suggestion
        uint16_t bandwidth = 0;
```

Reply:
Ok